src/control/drpc/modules.go

```diff
@@ -49,59 +53,85 @@ const (
 	MethodRequestCredentials = C.DRPC_METHOD_SEC_AGENT_REQUEST_CREDS
 )
 
+// MgmtMethod is a type alias for a drpc mgmt method.
+type MgmtMethod int32
+
+func (mm MgmtMethod) String() string {
+	return map[MgmtMethod]string{
+		MethodPrepShutdown: "prep shutdown",
+		MethodPingRank:     "ping",
+		MethodSetRank:      "set rank",
+		MethodSetUp:        "setup MS",
+	}[mm]
```

Comment:
This is fine and I don't object to adding a nice string representation for every method, but I also wouldn't insist on it, as it's a lot of work and I'm not sure it's used for anything other than debug statements. The main thing I wanted from the type alias was allowing the compiler to help with minimizing the possibility of incorrect int32 values being used instead of valid dRPC methods.